apps/desktop-wallet/src/api/apiDataHooks/utils/useFetchFtList.ts

```diff
@@ -35,17 +34,18 @@ type FTListProps = {
 }
 
 const useFetchFtList = (props?: FTListProps): FTList => {
-  const networkId = useAppSelector((s) => s.network.settings.networkId)
+  const networkId = useAppSelector(selectCurrentlyOnlineNetworkId)
   const network = networkId === 0 ? 'mainnet' : networkId === 1 ? 'testnet' : undefined
 
   const { data, isLoading } = useQuery({
-    queryKey: [TOKEN_LIST_QUERY_KEY, network],
+    queryKey: ['tokenList', network],
+    staleTime: ONE_DAY_MS,
+    gcTime: Infinity,
```

Comment:
I added `gcTime` in every query so that it's clear and so that we don't have to remember defaults.